### DISCUSSION TITLE
Monthly Roundup - July 2022

### DISCUSSION THREAD

**Discussion by tuves** [community/community#27917] [Announcements]
## **Monthly Community Roundup** ##

![Monthly Roundup Banner 2 0](https://user-images.githubusercontent.com/49915749/182164243-3f249369-ddb1-4ddc-9bb2-2a1a4216e067.png)

**ICYMI**

**_Our Community Managers Check-ins_**: community findings, cool ideas, things you should know, etc. 

Every few weeks, our Community Managers will check-in with folks to share patterns, interesting discussions, community member shoutouts, GitHub news, and more. Here's who you can expect these posts and where to find them:

  - @Akash1134 & @thomasshaped in [Actions and Packages](https://github.com/orgs/community/discussions/categories/actions-and-packages)
  - @queenofcorgis in [New to GitHub](https://github.com/orgs/community/discussions/categories/new-to-github), like this one [here](https://github.com/orgs/community/discussions/27492) where she shared the big moving 🏡 news! Check it out!

**_YouTube shorts and snacks_**:
  - [Take your planning & tracking to the next level. GitHub Projects now has a side panel view #Shorts](https://www.youtube.com/shorts/RfEyC8oRLYE)
  - [The Download: GitHub Copilot Goes GA, Deno Fresh, GPT-3 Code Explanations and more](https://www.youtube.com/watch?v=aE7xCkSeA1k)
  - [Configure version updates in Dependabot #shorts](https://www.youtube.com/shorts/Qcq0WfXz-aM)
  - [Front-end employee of the month: Copilot #Shorts](https://www.youtube.com/shorts/_3xgXhckrX4)
  - [Deploy to GitHub Pages with Custom GitHub Actions](https://www.youtube.com/shorts/Kq28yBigDYw)

**_Bigger Bites_**
  - [The Time Travel Debugger for Web Development](https://www.youtube.com/watch?v=N0eeYkpCv9o)
  - [GitHub Korea](https://www.youtube.com/watch?v=ujIRt14Qdfk)
  - [Introducing GitHub Skills](https://github.blog/2022-06-06-introducing-github-skills/)

**_Hot Off the (Product) Press_**

- [GitHub Sponsors available in 30 new regions](https://github.blog/2022-07-28-github-sponsors-available-in-30-new-regions-2/) - authored by our pal, @jlord
- [Tips & tricks for using GitHub Projects for personal productivity](https://github.blog/2022-07-21-tips-tricks-for-using-github-projects-for-personal-productivity/)
- [Research: How GitHub Copilot helps improve developer productivity](https://github.blog/2022-07-14-research-how-github-copilot-helps-improve-developer-productivity/)
- [Planning next to your code – GitHub Projects is now generally available](https://github.blog/2022-07-27-planning-next-to-your-code-github-projects-is-now-generally-available/)

Thank you for reading our July Monthly Roundup! Feel free to leave comments below. 👋🏽

**Comment by zhaomingzhan** [comment]


**Reply by 2118998** [reply]
> 你好怎么联系你

Talking

**Comment by Azizkhan1122** [comment]
Please help me I'm helping poor people all woorld

**Comment by Azizkhan1122** [comment]
Please help me

**Comment by anxumalo** [comment]
Hey @tuves Great article - thanks for sharing! I know this is an old announcement, but I have a suggested fix for one of your links in the **YouTube shorts and snacks** section to prevent the 404 error on YouTube. The link below appears as
`https://www.youtube.com/shorts/pxPIZD66FFc%5D(https:/www.youtube.com/watch?v=aE7xCkSeA1k%29`:

> [The Download: GitHub Copilot Goes GA, Deno Fresh, GPT-3 Code Explanations and more]([https://www.youtube.com/shorts/pxPIZD66FFc](https://www.youtube.com/watch?v=aE7xCkSeA1k)](https://www.youtube.com/shorts/pxPIZD66FFc%5D(https://www.youtube.com/watch?v=aE7xCkSeA1k)

So, based on the link text, the correct link for the video is: 

[The Download: GitHub Copilot Goes GA, Deno Fresh, GPT-3 Code Explanations and more](https://www.youtube.com/watch?v=aE7xCkSeA1k)

I don't know if you have a preferred link to a `/shorts/` video. Also, I'm not sure if you correct these so long after publishing, so please feel free to "use it or lose it" 😄 !!!

HTH

**Comment by tuves** [comment]
Thank you for the catch, @anxumalo! I've updated the link :)

**Comment by novium2** [comment]
Retiring this post as it is over 6 months old. If you have any questions, consider starting a [new post](https://github.com/orgs/community/discussions/new/choose).
